encoder.go

```diff
@@ -15,7 +16,8 @@ const (
 )
 
 var (
-	textEncode encoder = new(textEncoder)
+	textEncode     encoder = new(textEncoder)
+	timeZoneRegexp         = regexp.MustCompile("\\\\'.+\\\\'")
```

Comment:
please give an example, which string will be parsed by this pattern.

Reply:
seems like it will be faster to get text enclosed by  '\'  via string.find and slicing.

Reply:
The idea was to find substring of kind "\'<TIME_ZONE_NAME>\'" in string such as "DateTime(\'Europe/Moscow\')".
Yes, you're right, usage of find operations is much faster and enables to remove unquoting below.